src/components/pickers/Menu/Menu.stories.tsx

```diff
@@ -88,6 +90,35 @@ export const Default = ({ ...props }) => {
   );
 };
 
+export const InsideModal = () => {
```

Comment:
I think we need a play function for the story to make sure everything works

Reply:
Good poin. Yes

Reply:
Done. Added play test